samples/snippets/csharp/new-in-7/new-in-7/Program.cs

```diff
@@ -0,0 +1,185 @@
+﻿using static System.Console;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace new_in_7
+{
+    class Numerics
+    {
+        #region 32_BinaryConstants
+        public const int One =  0b0001;
+        public const int Two =  0b0010;
+        public const int Four = 0b0100;
+        public const int Eight = 0b1000;
+        #endregion
+
+        #region 33_ThousandSeparators
+        public const int Sixteen =   0b0001_0000;
+        public const int ThirtyTwo = 0b0010_0000;
+        public const int SixtyFour = 0b0100_0000;
+        public const int OneHundredTwentyEight = 0b1000_0000;
+        #endregion
+
+        #region 34_LargeIntegers
+        public const long BillionsAndBillions = 100_000_000_000;
+        #endregion
+
+        #region 35_OtherConstants
+        public const double AvogadroConstant = 6.022_140_857_747_474e23;
+        public const decimal GoldenRatio = 1.618_033_988_749_894_848_204_586_834_365_638_117_720_309_179M;
+        #endregion
+
+    }
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            OutVariableDeclarations();
+            TupleDeclarations();
+            TupleMethod();
+
+            #region 12_DeconstructPoint
+            var p = new Point(3.14, 2.71);
+            (double X, double Y) = p;
+            #endregion
+            WriteLine(X);
+            WriteLine(Y);
+
+            #region 13_DeconstructNames
+            (double horizontalDistance, double verticalDistance) = p;
```

Comment:
One variation of deconstruct that I didn't see in the samples is the `var` case.  It's possible to deconstruct `Point` as such:

``` csharp
var (x, y) = p;
```

Wanted to make sure you were aware of that feature.